src/DataManagement/MonthlyReport/MonthlyReportCalculator.cs

```diff
@@ -26,19 +26,18 @@ public class MonthlyReportCalculator
 
         private DateTime StartTime { get; set; }
         private DateTime EndTime { get; set; }
-        public CurrencyConverter CurrencyConverter { get; set; }
+        private CurrencyConverter CurrencyConverter { get; set; }
 
-        public MonthlyReportCalculator(IGoalDatabase goalDatabase, EntryDatabase entryDatabase, int userId,
-            CurrencyConverter currencyConverter)
+        public MonthlyReportCalculator(IGoalDatabase goalDatabase, EntryDatabase entryDatabase, CurrencyConverter currencyConverter)
```

Comment:
So when this itself is included in Startup it has access to other DI objects?

Reply:
Yes, so we don't have to create this manually in the controller. However I did have to change the calculate method to pass the user id parameter there